core/assets/jsons/levels/enemySurroundingPlayer.json

```diff
@@ -163,19 +163,19 @@
   "walls": [
     {
       "pos": [
-        10.88,
+        13.44,
         19.84
       ],
       "size": [
-        32,
+        26.88,
         1.28
       ],
       "texture": "wall-side"
     },
     {
       "pos": [
         0.64,
-        11.52
+        8.96
```

Comment:
Why these changes?

Reply:
Noticed in playtesting that there was a lot of dead space in the level. Removed it